internal/go/skycfg/yaml.go

```diff
@@ -86,8 +86,19 @@ func fnYamlUnmarshal(t *starlark.Thread, fn *starlark.Builtin, args starlark.Tup
 
 // toStarlarkValue is a DFS walk to translate the DAG from go to starlark
 func toStarlarkValue(obj interface{}) (starlark.Value, error) {
+	if obj == nil {
+		return starlark.None, nil
+	}
 	rt := reflect.TypeOf(obj)
 	switch rt.Kind() {
+	case reflect.Int:
```

Comment:
We also need to handle `reflect.Int64`, which `yaml.Unmarshal()` will use for integers in the range `(math.MaxInt32, math.MaxInt64]`: https://play.golang.org/p/aPb8Ml_cYSB

I'm also a little nervous about being so precise about which integer types the YAML parser returns. Can this code instead handle all five signed integer kinds in the same `case` and pass to `starlark.MakeInt64()`?